content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -166,151 +152,174 @@ A Deployment's rollout is triggered if and only if the Deployment's pod template
+  ```
+  The output is similar to this:  
+  ```
+  Name:                   nginx-deployment
+  Namespace:              default
+  CreationTimestamp:      Thu, 30 Nov 2017 10:56:25 +0000
+  Labels:                 app=nginx
+  Annotations:            deployment.kubernetes.io/revision=2
+  Selector:               app=nginx
+  Replicas:               3 desired | 3 updated | 3 total | 3 available | 0 unavailable
+  StrategyType:           RollingUpdate
+  MinReadySeconds:        0
+  RollingUpdateStrategy:  25% max unavailable, 25% max surge
+  Pod Template:
+    Labels:  app=nginx
+     Containers:
+      nginx:
+        Image:        nginx:1.9.1
+        Port:         80/TCP
+        Environment:  <none>
+        Mounts:       <none>
+      Volumes:        <none>
+    Conditions:
+      Type           Status  Reason
+      ----           ------  ------
+      Available      True    MinimumReplicasAvailable
+      Progressing    True    NewReplicaSetAvailable
+    OldReplicaSets:  <none>
+    NewReplicaSet:   nginx-deployment-1564180365 (3/3 replicas created)
+    Events:
+      Type    Reason             Age   From                   Message
+      ----    ------             ----  ----                   -------
+      Normal  ScalingReplicaSet  2m    deployment-controller  Scaled up replica set nginx-deployment-2035384211 to 3
+      Normal  ScalingReplicaSet  24s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 1
+      Normal  ScalingReplicaSet  22s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 2
+      Normal  ScalingReplicaSet  22s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 2
+      Normal  ScalingReplicaSet  19s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 1
+      Normal  ScalingReplicaSet  19s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 3
+      Normal  ScalingReplicaSet  14s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 0
+    ```
+    Here you see that when you first created the Deployment, it created a ReplicaSet (nginx-deployment-2035384211)
+    and scaled it up to 3 replicas directly. When you updated the Deployment, it created a new ReplicaSet
+    (nginx-deployment-1564180365) and scaled it up to 1 and then scaled down the old ReplicaSet to 2, so that at
+    least 2 Pods were available and at most 4 Pods were created at all times. It then continued scaling up and down
+    the new and the old ReplicaSet, with the same rolling update strategy. Finally, you'll have 3 available replicas
+    in the new ReplicaSet, and the old ReplicaSet is scaled down to 0.
 
 ### Rollover (aka multiple updates in-flight)
 
-Each time a new deployment object is observed by the Deployment controller, a ReplicaSet is created to bring up
+Each time a new Deployment is observed by the Deployment controller, a ReplicaSet is created to bring up
```

Comment:
Is the Deployment resource updated/created?

Reply:
Good catch. It was actually giving the wrong meaning. I reworded the content.